service/src/test/java/com/epam/esm/service/impl/TagServiceImplTest.java

```diff
@@ -30,6 +31,7 @@
 
 
 @ExtendWith(MockitoExtension.class)
+//@ContextConfiguration(classes = {ServiceConfig.class, DaoConfig.class})
 public class TagServiceImplTest {
```

Comment:
"The service layer should be covered with unit tests not less than 80%"
I don't see any written and uncommented tests for OrderService, TagService, UserService, and all validators on the service layer. Please, cover that with unit tests

Reply:
Commits:
7052eef , a489597 , 3e267d4 , 8e3bcdd , 4613d14 , fb690f8 , 2f38272 , db324ad .